Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -46,8 +46,12 @@ def set_marshall_helper(WorkspaceId, Enabled, AzureDnsAgentSvcZone, mock_worker_
         try:
             if not os.path.isdir(WORKER_STATE_DIR):
                 os.mkdir(WORKER_STATE_DIR)
+            os.chmod(WORKER_STATE_DIR, FILE_PERMISSION_LEVEL)
```

Comment:
Minor : You might want to change "FILE_PERMISSION_LEVEL" to a more explicit name